.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: Deploy Jenkins jobs
+on:
+  push:
+    branches:
+      - main
+jobs:
+  push:
+    name: Deploy Jenkins jobs
+    runs-on: ubuntu-latest
+    if: github.repository == 'gluster/centosci'
```

Comment:
Why do we need this repository check?

Reply:
so that if someone forks the repository, this action does not run automatically (it will fail without the `JENKINS_API_KEY` secret)

Reply:
Ah..got it